docs/source/features/passes/onnx.md

```diff
@@ -79,6 +79,103 @@ Please refer to [OrtTransformersOptimization](ort_transformers_optimization) for
+
+Here are some examples to describe the pre/post processing which is exactly same with [superresolution](https://github.com/microsoft/onnxruntime-extensions/blob/main/onnxruntime_extensions/tools/add_pre_post_processing_to_model.py#L89)
+
+```json
+{
+    "pre": [
+        {"ConvertImageToBGR": {}},
+        {
+            "Resize": {
+                "resize_to": [
+                    {"type": "__model_input__", "input_index": 0, "dim_index": -2},
+                    {"type": "__model_input__", "input_index": 0, "dim_index": -1},
+                ]
+            }
+        },
+        {
+            "CenterCrop": {
+                "height": {"type": "__model_input__", "input_index": 0, "dim_index": -2},
+                "width": {"type": "__model_input__", "input_index": 0, "dim_index": -1},
+            }
+        },
+        {"PixelsToYCbCr": {"layout": "BGR"}},
+        {"ImageBytesToFloat": {}},
+        {"Unsqueeze": {"axes": [0, 1]}},
+    ],
+    "post": [
+        {"Squeeze": {"axes": [0, 1]}},
+        {"FloatToImageBytes": {"name": "Y1_uint8"}},
+        {
+            "Resize": {
+                "params": {
+                    "resize_to": [
+                        {"type": "__model_output__", "output_index": 0, "dim_index": -2},
+                        {"type": "__model_output__", "output_index": 0, "dim_index": -1},
+                    ],
+                    "layout": "HW",
+                },
+                "io_map": [["PixelsToYCbCr", 1, 0]],
+            }
+        },
+        {"FloatToImageBytes": {"multiplier": 1.0, "name": "Cb1_uint8"}},
+        {
+            "Resize": {
+                "params": {
+                    "resize_to": [
+                        {"type": "__model_output__", "output_index": 0, "dim_index": -2},
+                        {"type": "__model_output__", "output_index": 0, "dim_index": -1},
+                    ],
+                    "layout": "HW",
+                },
+                "io_map": [["PixelsToYCbCr", 2, 0]],
+            }
+        },
+        {"FloatToImageBytes": {"multiplier": 1.0, "name": "Cr1_uint8"}},
+        {
+            "YCbCrToPixels": {
+                "params": {
+                    "layout": "BGR",
+                },
+                "io_map": [
+                    ["Y1_uint8", 0, 0],
+                    ["Cb1_uint8", 0, 1],
+                    ["Cr1_uint8", 0, 2],
+                ],
+            }
+        },
+        {"ConvertBGRToImage": {"image_format": "png"}},
+    ],
+    "tool_command_args": [
+        {
+            "name": "image",
+            "data_type": "uint8",
+            "shape": ["num_bytes"],
+        }
+    ],
+    "target_opset": 16,
+}
+```
+
+* Olive introduce two placeholders to represent the model input/ouput shape dimension value: `__model_input__` and `__model_output__`.
```

Comment:
`introduces`.

Also, could you help put this line before the example? I was confusing what is the meaning of `__model_input__` when I read the example before this line.

Reply:
will do